src/cordova/www/css/index.css

```diff
@@ -1,5 +1,62 @@
-﻿body {
-    -webkit-touch-callout: none;
-    -webkit-text-size-adjust: none;
-    -webkit-user-select: none;
-}
\ No newline at end of file
+    body {
```

Comment:
This wouldn't be the right file to add this. Instead you want to add it to src/client/content/styles.css, then run `npm run build-cordova`, then run `src\copybundles.cmd`, to get it to the cordova app folder